staging/src/github.com/kubeedge/mapper-framework-Java/src/main/java/grpc/GrpcServer.java

```diff
@@ -0,0 +1,210 @@
+package grpc;
+
+import devicepanel.DevPanel;
+import devicepanel.Device;
+import dmi.v1beta1.Api;
+import dmi.v1beta1.DeviceManagerServiceGrpc;
+import dmi.v1beta1.DeviceMapperServiceGrpc;
+import io.grpc.Server;
+import io.grpc.netty.NettyServerBuilder;
+import io.grpc.stub.StreamObserver;
+import io.netty.channel.epoll.EpollEventLoopGroup;
+import io.netty.channel.epoll.EpollServerDomainSocketChannel;
+import io.netty.channel.unix.DomainSocketAddress;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import model.CustomizedDev;
+import model.DeviceInstance;
+import model.DeviceModel;
+
+import java.io.IOException;
+import java.util.List;
+
+import static devicepanel.Device.buildApiTwinsFromLocal;
+
+@Slf4j
+@Getter @Setter
+public class GrpcServer extends DeviceManagerServiceGrpc.DeviceManagerServiceImplBase {
+    private Server grpcServer;
+    private ConfigGrpcServer cfg;
+    private DevPanel devPanel;
+//    private UnixServerSocketChannel serverSocket;
+
+    @Getter @Setter
+    public static class ConfigGrpcServer{
+        private String socketPath;
+        private String protocol;
+
+        public ConfigGrpcServer(String socketPath, String protocol) {
+            this.socketPath = socketPath;
+            this.protocol = protocol;
+        }
+    }
+
+    public GrpcServer(ConfigGrpcServer cfg, DevPanel devPanel){
+        this.cfg = cfg;
+        this.devPanel = devPanel;
+    }
+
+    public void start() throws IOException {
+//        String socketPath = cfg.getSocketPath();
+//
+//        File socketFile = new File(socketPath);
+//        if (socketFile.exists()) {
+//            if (socketFile.isDirectory()) {
```

Comment:
Annotate the code, add comments to explain why, or simply delete them and explain in the readme document

Reply:
OK, I will correct it.